### DISCUSSION TITLE
Why onSelectionContextMenu is not documented?

### DISCUSSION THREAD

**Discussion by goliney** [xyflow/xyflow#3760] [General]
I was surprised to find out there is no documentation for `onSelectionContextMenu` prop of `<ReactFlow />`.

<img width="665" alt="image" src="https://github.com/xyflow/xyflow/assets/2017344/111cceb4-c4e1-4886-ab02-9fecf760f1a1">

I'm usually hesitant to use undocumented features. Is there a reason why this feature is not documented?

**Comment by bcakmakoglu** [comment]
That's a good question, I don't think there's a reason this isn't documented, it's not experimental or anything 😄 

It's just missing from the docs from what I can tell.
If you'd like to contribute you could open a PR to add it to the docs, [the relevant file is this one](https://github.com/xyflow/web/blob/main/sites/reactflow.dev/src/page-data/reference/ReactFlow.props.ts).
